packages/admin/src/targetGroups/allAssignedContacts/AssignedContactsGrid.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable @comet/no-private-sibling-import */
```

Comment:
This rule should not be disabled. Why is this neccessary here?

Reply:
Thank you! 
I used the wrong import here.